.idea/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/claim-app-data.iml" filepath="$PROJECT_DIR$/.idea/claim-app-data.iml" />
```

Comment:
What are these changes?

Reply:
The `.idea` folder is from jetbrains IDEs and was pushed sometime in the past and now seems to have been updated by my IDE. We need to remove the folder from git in order to prevent changes but I would do this separately.